contributor-guide/modules/ROOT/pages/version-control.adoc

```diff
@@ -7,51 +7,125 @@ Boost uses the Git version control system for its development. This allows multi
+* The implications of merging something in your library or submodule to the Super-project *develop* branch, include that due to how Boost is tested, you may affect other developers. As part of Boost testing, Continuous Integration (CI) usually clones the Super-project and any required submodules. 
 
-### Version of <Library/Boost>
+** When testing any branch that is _not_ *master*, the *develop* branch of the Super-project is cloned. 
+** When you're testing *master* or a release, the *master* branch of the Super-project is cloned. 
 
-_If you can find the version of the library itself (in a readme file, other documentation, or a header file), then put that version number here. Otherwise, if you have difficulty finding a version for the library itself, add the Boost version number, such as `1.82.0`._
+While this cloning process is not a _requirement_, most libraries follow it.
 
-### Steps necessary to reproduce the problem
+Merging something to develop may affect any dependent code in the rest of Boost as soon as the commit bot makes the update to the Super-project. For example, if a mistake is made to the MySQL library *develop* branch, then it will affect the CMake module, because this module runs tests to verify that you can use MySQL via CMake.
 
-_A small compiling program is the best. If your code is public, you can provide a link to the repository._
+NOTE:: Developers don't directly create Tags under repositories, when working with Boost. Tags are created by the release scripts, both in the Super-project and the individual repos, so do not add any of your own. This differs from the usual workflow in other non-Boost projects.
++
+If you need to look at the code for the 1.82.0 release, navigate to the relevant repo, and enter:
++
+[source,text]
+----
+git checkout boost-1.82.0 
+----
 
-### All relevant compiler information
+== Breaking Changes
 
-_If you are unable to compile include the type and version of compiler you are using as well as all compiler output including the error message, file, and line numbers involved._
+Boost maintains its release versions with https://github.com/boostorg/boost/tags[Tags], where the tag is always the `boost-` prefix followed by a https://semver.org/[Semantic Versioning] string.  Using semver strings is useful because that's something most tools understand, and for most users who just want to find a certain boost version, that's enough information.
 
-The more information you provide the sooner your issue can get resolved.
+But for users who want to know if something broke from one version to another, that's where things get complicated. The semver specification  requires an increase in the major version number (for example, from `1.2.3` to `2.0.0`) if something breaks the API. The minor version should be increased (for example: `1.81.0` to `1.82.0`) if there are new features.
```

Comment:
Expressions like "to know if something broke" looked OK for a quick message but are a little too informal to be in the documentation.

Reply:
agreed, reworded